### ISSUE TITLE
Support ESLint object-curly-spacing rule for formatting

### ISSUE THREAD

**Issue by cmlenz** [daidodo/format-imports-vscode#68]
I have ESLint set up to not have spaces inside curlies, using the `object-curly-spacing` rule as follows:

```
rules: {
  'object-curly-spacing': ['warn', 'never'],
}
```

Since version 7.4.13 the import sorter extension will reformat my imports but put the spaces inside the curlies. For whatever reason, this works fine in 7.4.10.

It would be nice if the [object-curly-spacing](https://eslint.org/docs/rules/object-curly-spacing) rule was honored by the import sorter extension.

This was originally raised in #67

**Comment by daidodo** [comment]
Thanks for the feedback!

In short-term, you can set [bracketSpacing](https://github.com/daidodo/format-imports/blob/main/docs/interfaces/Configuration.md#bracketspacing) before next release:
```json
{
  "bracketSpacing": false
}
```

**Comment by cmlenz** [comment]
Thanks @daidodo , that works, even though for some reason that option is offered by the VSCode auto complete or the settings UI.

Also, a big thank you for this extension in general, it's such an important part of my coding workflow, but you only ever really notice when it's not doing its magic :)

**Comment by daidodo** [comment]
Yes, auto-completion is part of the plugin as well.

I've released v7.4.14 to address this issue. With it you can remove the above `bracketSpacing` config.
Thanks very much!
